docs/_static/js/expand_tabs.js

```diff
@@ -14,4 +18,20 @@ $( document ).ready(function() {
       tab.click();
     }
   }
+
+  // Uses a hash referencing a Sphinx label from the URL page#sphinx-label
+  var hash = window.location.hash.substr(1);
+  hash = hash.replace(/[^0-9a-z\-_]/gi, '');
+  // If the hash is inside a tab panel
+  var tab_with_reference = $(".sphinx-tabs-panel #" + hash).parents(".sphinx-tabs-panel");
+
+  if (tab_with_reference.length > 0) {
+    // Use the panel's ID to guess the tab's ID
+    var panel_id = tab_with_reference.first().attr("id");
+    var tab_id = panel_id.replace("panel-", "tab-");
+    // Invoke the tab buttons click() method to display it
+    $("button#" + tab_id).click();
+    // Scroll the tab widget into view
+    $('html, body').animate({ scrollTop: tab_with_reference.parents(".sphinx-tabs").first().offset().top}, 1000);
+  }
```

Comment:
@stsewd I refactored previous sections (GitHub/Bitbucket/GitLab/Gitea) into tabs. Because each subsection had a Sphinx label, I moved the label inside the tab and wrote some JS that will automatically display the tab. It seems to work well :+1:

Reply:
You can test it by noticing what happens with this link: https://docs--9675.org.readthedocs.build/en/9675/guides/git-integrations.html#webhook-integration-bitbucket

`webhook-integration-bitbucket` is the Sphinx label.

Reply:
This seems like a good solution. I thought we already had JS to display the tab?

Reply:
Yes, we do - this was to keep the currently existing Sphinx labels, in case there's an intersphinx reference or some other Stackoverflow reference. In that case, they will keep working :+1:

Reply:
Also, this makes it generally possible to swap a section with a tab and vice-versa.

Reply:
It also has a funny animated scrolling feature... I actually had to add _something_ because the normal anchor went too far down for the tab menu to be visible. So it ended up being this version.